js/data/odata/store.js

```diff
@@ -195,21 +186,17 @@ const ODataStore = Store.inherit({
         };
 
         // NOTE: For AppBuilder, do not remove
-        if(isDefined(loadOptions.urlOverride)) {
-            url = loadOptions.urlOverride;
-        } else {
-            url = this._url;
-        }
+        url = loadOptions.urlOverride ?? this._url;
```

Comment:
```suggestion
        url = loadOptions && loadOptions.urlOverride ?? this._url;
```

Or if we enabled optional chaining as well as coalescing:
```suggestion
        url = loadOptions?.urlOverride ?? this._url;
```

Reply:
Same for ` if(loadOptions.customQueryParams) {` below